test/Libraries/DynamoPythonTests/CodeCompletionTests.cs

```diff
@@ -293,6 +292,7 @@ public void CanMatchImportSystemLibraryWithComment()
             var matches = IronPythonCompletionProvider.FindBasicImportStatements(str);
 
             Assert.AreEqual(1, matches.Count);
+            Assert.IsTrue(matches.ContainsKey("System"));
```

Comment:
you could use sequence equal here to be more specific with the test results

Reply:
This was the only location we strictly checked the count without verifying the matching key didn't change.  Should I update this for all instances?